.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,42 @@
+name: Feature Request
+description:  Create a feature request for new features
+labels: ["Type: Feature Request", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for showing interest in adding new features to openlibrary! 👍
+
+  - type: textarea
+    attributes:
+      label: Problem
+      description: Describe the problem that this feature will address.
+      value: |
+        ### A clear and concise description of what you want to happen
+
+        ### Expected behaviour / screenshots (ex: Figma design screenshots for UI feature)
+
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Proposal & Constraints
+      description: Describe the proposed solution and possible constraints
+      value: |
+        ### What is the proposed solution / implementation?
+
+        ### Is there a precedent of this approach succeeding elsewhere?
+
+        ### Which suggestions or requirements should be considered for how feature needs to appear or be implemented?
+
+  - type: textarea
+    attributes:
+      label: Additional Context
```

Comment:
@cdrini  are you saying to combine Additional context with proposal & constraints, i.e add proposal & constraints inside additional context? I just want to confirm so that I can make changes accordingly.

Reply:
Thanks for clarification @RayBB. I will add the section for leads and tag you when I push the changes